### ISSUE TITLE
can't search and select from the list

### ISSUE THREAD

**Issue by afifisuha** [FACN4/stars_of_the_north_w4#20]
when i type a letter or a string i cant search through the proposed list or even click on it

**Comment by afifisuha** [comment]
and the search nav bar also is not working

**Comment by afifisuha** [comment]
yes but the nav bar still not working(when typing the full name i can click on it to take me to the related link)
but i cant click on it, however, when hitting enter after typing the whole name, the web responses with error meassage

**Comment by matthall8** [comment]
Let's talk about this more today.
